src/EventSubscriber/MenuSubscriber.php

```diff
@@ -81,6 +81,10 @@ private function configureMainMenu(SidebarMenuEvent $menu)
                 new MenuItemModel('export', 'menu.export', 'export', [], $this->getIcon('export'))
             );
         }
+
+        $menu->addItem(
```

Comment:
I'm not sure which permissions to require here. Do I need to define a new or use an existing one?

Reply:
why adding a new menu here? I would move it to the toolbar (action menu) of the project listing instead

Reply:
![image](https://user-images.githubusercontent.com/3410507/84603968-2d1e9700-ae69-11ea-839b-952ebe73743c.png)
Considering the image above, should I add it to the menu number 1 or 2?

Reply:
1 - see [here](https://github.com/kevinpapst/kimai2/blob/master/templates/project/actions.html.twig#L9)